tests/system/test_system_dbapi.py

```diff
@@ -378,6 +378,62 @@ def test_execute_many(self):
         self.assertEqual(res[0], 1)
         conn.close()
 
+    def test_DDL_autocommit(self):
+        """Check that DDLs in autocommit mode are immediately executed."""
+        conn = Connection(Config.INSTANCE, self._db)
+        conn.autocommit = True
+
+        cur = conn.cursor()
+        cur.execute(
+            """
+            CREATE TABLE Singers (
+                SingerId     INT64 NOT NULL,
+                Name    STRING(1024),
+            ) PRIMARY KEY (SingerId)
+        """
+        )
+        conn.close()
+
+        # if previous DDL wasn't committed, the next INSERT
+        # will fail with a ProgrammingError
+        conn = Connection(Config.INSTANCE, self._db)
+
+        cur = conn.cursor()
+        cur.execute("""INSERT INTO Singers (SingerId, Name) VALUES (1, "Name")""")
+
+        conn.commit()
+
+        cur.execute("DROP TABLE Singers")
```

Comment:
Wouldn't this statement fail if the `CREATE` statement wasn't executed? Could we remove the `INSERT`?

Reply:
Okay, pushed DROP TABLE into both cases